### PR TITLE
[Feature][Transforms-V2] LLM transforms Support KimiAI

### PR DESCRIPTION
**Pull request by hawk9821**
### Purpose of this pull request

LLM transforms Support KimiAI

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
E2E : TestLLMIT#testLLMWithKimiAI
UT:  LLMRequestJsonTest#testKimiAIRequestJson

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/transform-v2/llm.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ more.
 ### model_provider
 
 The model provider to use. The available options are:
-OPENAI、DOUBAO、CUSTOM
+OPENAI、DOUBAO、CUSTOM、KIMIAI
```
Comment (Hisoka-X) [inline docs/en/transform-v2/llm.md]:
```suggestion
OPENAI, DOUBAO, KIMIAI, CUSTOM
```

File: docs/en/transform-v2/llm.md [hunk 2]
```diff
@@ -155,14 +155,19 @@ The `custom_request_body` option supports placeholders:
 
 Transform plugin common parameters, please refer to [Transform Plugin](common-options.md) for details
 
-## Example
+## tips
+The API interface usually has a rate limit, which can be configured with Seatunnel's speed limit to ensure smooth operation of the task.
+For details about Seatunnel speed limit Settings, please refer to [speed-limit](../../zh/concept/speed-limit.md) for details.
```
Comment (Hisoka-X) [inline docs/en/transform-v2/llm.md]:
```suggestion
For details about Seatunnel speed limit Settings, please refer to [speed-limit](../concept/speed-limit.md) for details.
```

File: docs/zh/transform-v2/llm.md [hunk 3]
```diff
@@ -148,14 +147,19 @@ transform {
 
 转换插件的常见参数, 请参考  [Transform Plugin](common-options.md) 了解详情
 
-## 示例
+## tips
+大模型API接口通常会有速率限制，可以配合Seatunnel的限速配置，已确保任务顺利运行。
+Seatunnel限速配置,请参考[speed-limit](../../zh/concept/speed-limit.md)了解详情
```
Comment (Hisoka-X) [inline docs/zh/transform-v2/llm.md]:
```suggestion
Seatunnel限速配置,请参考[speed-limit](../concept/speed-limit.md)了解详情
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @hawk9821 !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**